openedx/features/classrooms/models.py

```diff
@@ -0,0 +1,13 @@
+from nodebb.models import DiscussionCommunity
+from model_utils.models import TimeStampedModel
```

Comment:
Please run `isort` on your files to fix import ordering.

Reply:
Done.